miniapps/performance/makefile

```diff
@@ -82,6 +82,10 @@ endef
 
 ifneq (YES,$(MFEM_USE_CUDA))
   PEDANTIC_FLAG = -pedantic
+else
+	ifeq (YES,$(MFEM_USE_SIMD))
+		MFEM_FLAGS := $(filter-out -x=cu, $(MFEM_FLAGS))
+	endif
```

Comment:
This seems redundant. Was there a specific reason to add it?

Reply:
I think I understand why you had to add this: without `-x=cu` when a `.cpp` file is compiled, `nvcc` does not define `__CUDACC__`, so you get the intrinsics definitions.

This is a convoluted way of enabling the intrinsics. Isn't there a more streamlined way of achieving this?

Reply:
Right - I agree this solution does not allow to have both `MFEM_FORALL` **and** `intrinsics` in the same file, which we actually dont have in `MFEM`.
The issue comes with `nvcc` including these headers, but won't ever use them.